understanding/20/use-of-color.html

```diff
@@ -38,13 +38,28 @@ <h2>Intent of Use of Color</h2>
          only in color.
       </p>
       
-      <p>Examples of information conveyed by color differences: “required fields are red",
-         “error is shown in red", and “Mary's sales are in red, Tom's are in blue". Examples
-         of indications of an action include: using color to indicate that a link will open
-         in a new window or that a database entry has been updated successfully. An example
-         of prompting a response would be: using highlighting on form fields to indicate that
-         a required field had been left blank.
-      </p>
+      <p>Examples of common failures include:</p>
+      <ul>
+        <li>
+            <strong>Information conveyed by color differences:</strong> 
+            <ul>
+                <li>required fields are only identified by changing their border to a red color;</li>
+                <li>the type of transaction is only visually distinguished by using different foreground colors: red for debits and green for credits;</li>
+                <li>subway lines are visually distinguished solely by employing various filled colors.</li>
+            </ul>
+        </li>
+        <li>using color alone to indicate
+         that a link will open in a new window or that a database entry has been updated successfully.</li>
+        <li>using subtle color highlighting on form fields to prompt for a response on 
```

Comment:
I know this was a pre-existing example, but I really question if this is a Use of Color failure, and I think we should remove it. If someone used grey to do that prompting (i.e., the background was white by default and it had a grey visual effect to prompt, I don't think that would be considered a Use of Color. A subtle grey highlight may not be a *great* prompt; I'm just saying I don't think it fails this SC.
You somewhat touch on this concept in the Note (which I also think may confuse some people). It is clear to me that the working group did NOT regard the appearance of new content as 'use of color'. Assuming a white background, ANY visually distinguished content (i.e., non-white) is theoretically a color, but clearly Use of Color is talking about the _change_ of color in pre-existing information (i.e., going from an absence of color to the appearance of color is NOT a Use of Color).  Until 2.1, we did not even have a measure for non-text contrast, so I also feel that Use of Color is biased to text -- and to visual elements that are considered part of text elements (such as the outline around a text input).

This entire area should really be more emphatically discussed in the Understanding document, and we should attempt to see if we can get working group consensus on it.